lang/python/setup_pip.py

```diff
@@ -329,23 +290,28 @@ def source_filter(sources):
         d = os.path.join(stage_dir, os.path.dirname(f))
         if not os.path.isdir(d):
             os.makedirs(d)
-        if f in movers:
-            src = movers[f]
-        else:
-            src = f
         # Symlinks are not followed in setup, we need to use real files.
-        shutil.copy2(src, os.path.join(stage_dir, f))
+        shutil.copy2(f, os.path.join(stage_dir, f))
+    # Copy files in lang/python/wiredtiger to the root folder.
+    pywt_path = os.path.join(stage_dir, "lang", "python", "wiredtiger")
+    pywt_files = os.listdir(pywt_path)
+    for f in pywt_files:
+        basename = os.path.basename(f)
+        src = os.path.join(pywt_path, f)
+        if basename == 'init.py':
+            shutil.copy2(src, os.path.join(stage_dir, '__init__.py'))
+        else:
+            shutil.copy2(src, os.path.join(stage_dir, basename))
```

Comment:
Would it be clearer to `shutil.copytree` the whole tree, and then selectively rename the `init.py` file?